vaadin-split-layout-flow/src/main/java/com/vaadin/flow/component/splitlayout/SplitLayout.java

```diff
@@ -349,9 +348,18 @@ public Registration addSplitterDragendListener(
         return super.addSplitterDragendListener(listener);
     }
 
-    private void setInnerComponentStyle(Component innerComponent,
-                                               String styleName, String value) {
-        Optional.ofNullable(innerComponent).ifPresent(component -> component
-                .getElement().getStyle().set(styleName, value));
+    private void setInnerComponentStyle(String styleName, String value,
+            boolean primary) {
+        Component innerComponent = primary ? primaryComponent
+                : secondaryComponent;
+        if (innerComponent != null) {
+            innerComponent.getElement().getStyle().set(styleName, value);
+        } else {
+            getElement().executeJs(
+                    "var children = this.children;"
+                            + "var element = children && children[$0];"
```

Comment:
nit: Unnecessary check. `this.children` is always available (as long as `this` is an element)

Reply:
agreed, was being unnecessarily careful to avoid possible js execution exceptions, simplified